inputfiles/idlSources.json

```diff
@@ -276,6 +276,10 @@
         "url": "https://html.spec.whatwg.org/multipage/links.html",
         "title": "HTML - Links"
     },
+    {
+        "url": "https://html.spec.whatwg.org/multipage/media.html",
+        "title": "HTML - Media"
+    },
```

Comment:
This is the controversial one: It's only supported in Safari and old edge today (without flags ) according to caniuse:
 - https://caniuse.com/#search=video%20track
- https://caniuse.com/#search=audio%20track

Reply:
Ah, didn't know that, nice find! But those types are already in lib.d.ts so removing them might affect existing users. Should I remove them anyway?

Reply:
We think of it as the status quo is wrong, yep, but adding the new spec would make people expect something exists in the runtime which doesn't (without turning on a flag for 90% of the web). 

If it gets removed from  the PR is it still feasible for the media playback to come in?  Then we keep the dodgy existing behavior until the runtime features available for everyone

Reply:
The spec itself also covers an essential part of HTML - `HTMLAudio/Video/TrackElement`, so I think the spec should still be added, while we can drop specific types mentioned here.

>If it gets removed from the PR is it still feasible for the media playback to come in?

It shouldn't be affected, as it only adds a function.

Reply:
Yep, I agree - let's do that 👍